solutions/swb-reference/src/services/dataSetService.ts

```diff
@@ -144,13 +154,27 @@ export class DataSetService implements DataSetPlugin {
   ): Promise<PaginatedResponse<DataSet>> {
     let projectDatasets: DataSet[] = [];
 
+    if (user.roles.includes('ITAdmin')) {
+      return {
+        data: [],
+        paginationToken: undefined
+      };
+    }
+
+    let page = 0;
     let lastPaginationToken = paginationToken;
     do {
-      const allDatasets = await this._workbenchDataSetService.listDataSets(user, pageSize, paginationToken);
+      console.log(`page ${page}`);
+      page += 1;
```

Comment:
Not blocking at this point, but we should remember to g oin and remove this debugging stuff.